editor.planx.uk/src/pages/FlowEditor/lib/__tests__/filters.test.ts

```diff
@@ -1,163 +1,80 @@
-import type { Store } from "../store";
 import { vanillaStore } from "../store";
+import flowWithBranchingFilters from "./mocks/flowWithBranchingFilters.json";
+import flowWithSingleFilter from "./mocks/flowWithSingleFilter.json";
 
 const { getState, setState } = vanillaStore;
+const { upcomingCardIds, resetPreview, record, currentCard, collectedFlags } =
+  getState();
 
 // https://i.imgur.com/k0kkKox.png
+describe("Single filter", () => {
```

Comment:
naming nit: rather than "Single filter", can we please refer to this as "Root filter"? as i understand this bug, it's not actually about the number of the filter type nodes, but rather their placement in the graph (eg directly under `_root` or on a branch). Even if I had multiple filters that were all direct edges of `_root`, they should still behave as expected!

Reply:
This is correct - I'll make the change.

I originally had it as "single filters" and "multiple filters" - I redefined the latter but not the original 👍